packages/core/src/shipping/index.ts

```diff
@@ -3,7 +3,7 @@ export * from './shipping-request-options';
 
 export { default as createShippingStrategyRegistry } from './create-shipping-strategy-registry';
 
-export { default as Consignment, ConsignmentsRequestBody, ConsignmentRequestBody, ConsignmentAssignmentRequestBody, ConsignmentUpdateRequestBody } from './consignment';
+export { default as Consignment, ConsignmentsRequestBody, ConsignmentRequestBody, ConsignmentAssignmentRequestBody, ConsignmentUpdateRequestBody, ConsignmentLineItem } from './consignment';
```

Comment:
Is this export used somewhere, I had a look in the PR but couldn't find usage?

Reply:
removed